internal/network/machine_network_cidr.go

```diff
@@ -126,6 +134,11 @@ func VerifyVip(hosts []*models.Host, machineNetworkCidr string, vip string, vipN
 	if ipIsBroadcast(vip, machineNetworkCidr) {
 		return models.VipVerificationFailed, errors.Errorf("%s <%s> is the broadcast address of machine-network-cidr <%s>", vipName, vip, machineNetworkCidr)
 	}
+
+	if !verifyVipFree {
```

Comment:
Consider renameing to `shouldVerifyIfVipIsFree`, current name is too similar to  `VerifyVipFree`

Reply:
Agree